default.nix

```diff
@@ -55,12 +55,11 @@ stdenv.mkDerivation rec {
     antlr4
     ocamlPackages.odoc
   ]
-  ++ optionals doInstallCheck (
+  ++ optionals doInstallCheck [
     # Test-suite dependencies
     # ncurses is required to build an OCaml REPL
-    optional (!versionAtLeast ocaml.version "4.07") ncurses
-    ++ [ ocamlPackages.ounit rsync which ]
-  )
+    ncurses ocamlPackages.ounit rsync which
```

Comment:
This conditional was the other way around: ncurses is only needed if OCaml < 4.07 is used. So it shouldn't appear in the list now.